Tutorial/Window/wh_tl_co_011196.csv

```diff
@@ -1,5 +1,5 @@
 name01#0,"""The Matter Board"""
-text01#0,"""Matter Board<br><br>By completing designated objectives, the Matter Board allows new paths to be unlocked.<br><br>There are many objectives, such as defeating enemies, picking up item drops, or meeting specific people.<br><br>By completing these objectives, new paths continue and the story progresses. Continue these at whatever pace you like."""
```

Comment:
They completely redid the MB tutorial for ep 4 so this old translation was no longer correct.